changelog/v1.27.5-patch2/base64url.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+- type: FIX
+  issueLink: https://github.com/solo-io/envoy-gloo/issues/329
+  resolvesIssue: false
+  description: >-
+    Fall back to attempting to decode Base64Url if Base64 decoding fails in the Inja transformation
```

Comment:
according to the linked issue, "the transformation fails silently because the decode returned an empty string" - while we're at this, is it possible to add a warning of some sort when this happens?

Reply:
I think thats out of scope for this pr as its a problem with many of our transformations and has been in general the accepted state.

Some customers may rely on this today